config/locales/pages/data_sharing_agreement/en.yml

```diff
@@ -0,0 +1,567 @@
+          where appropriate. Such arrangements will include (but will
+          not be limited to):
+        </p>
+        <ul class='govuk-list govuk-list--bullet'>
+          <li>containing the incident and mitigating any ongoing risk</li>
+          <li>recovering information</li>
+          <li>assessing whether the Information commissioner should be
+            notified or whether data protection officers or data
+            subjects need to be notified</li>
+        </ul>
+        <p class='govuk-body'>
+          The arrangements may vary in each case, depending on the
+          sensitivity of the information and the nature of the loss or
+          unauthorised disclosure. Where appropriate and if relevant to
+          the incident, disciplinary misconduct action and criminal
+          proceedings will be considered.
+        </p>
+        <h3 class='govuk-heading-m'>
+          Consequences of security incidents
+        </h3>
+        <p class='govuk-body'>
+          Any security incident that occurs will not impact the DfE’s
+          ongoing cooperation with the provider.
+        </p>
+        <p class='govuk-body'>
+          In the event of a personal data breach (or where there is a
+          reason to believe that an incident could arise) the DfE and
+          the provider will pause further data sharing until the
+          signatories of this agreement are satisfied that the cause or
+          incident is resolved.
+        </p>
+        <p class='govuk-body'>
+          The DfE contact would raise the incident with the DfE
+          departmental security team and complete formal procedures in
+          the event of a personal data breach.
+        </p>
+        <h2 class='govuk-heading-m' id='#termination-of-the-agreement'>
+          Termination of the agreement
+        </h2>
+        <h3 class='govuk-heading-s'>
+          When to terminate this agreement
+        </h3>
+        <p class='govuk-body'>
+          Both participants to this DSA reserve the right to terminate
+          this DSA with 3 months’ notice in the following circumstance:
+        </p>
+        <ul class='govuk-list govuk-list--bullet'>
+          <li>
+            finances, resources or other factors beyond the control of
+            the DfE or the provider
+          </li>
+          <li>
+            if any change occurs which, in the opinion of the DfE and
+            the provider, significantly impairs the value of
+            the data sharing arrangement in meeting their objectives
+          </li>
+          <li>
+            in the event of a significant security breach or other
+            serious breach of the terms of this DSA by either
+            participant, the DSA will be terminated or suspended
+            immediately without notice
+          </li>
+        </ul>
+        <h2 class='govuk-heading-m' id='#your-consent'>
+          Your consent
+        </h2>
+        <p class='govuk'>
```

Comment:
I think maybe this class should be `class='govuk-body'` as formatting looks a little weird locally.

Reply:
![Screenshot 2022-04-27 at 14 48 40](https://user-images.githubusercontent.com/43522239/165533686-96ec3cf0-e538-4edb-8410-f94ad9d98c78.png)